### PR TITLE
Non-binary incidence

### PR DESCRIPTION
**Pull request by jamiecook**
Hey @bstabler, we were looking to do some work using PopSim and noticed that the non-binary incidence hadn't made its way into a release.  So I generated the PR :)

### CODE REVIEW

File: docs/software.rst [hunk 1]
```diff
@@ -234,3 +234,5 @@ Release Notes
   * v0.3.4 - add survey weighting use case
   * v0.3.5 - add Python 3.5+ support
   * v0.4 - transfer to ActivitySim.org
+  * v0.4.1 - package updates
+  * v0.4.2 - validation script in Python
```
Comment (jamiecook) [inline docs/software.rst:238]:
```suggestion
  * v0.4.2 - allow non-binary incidences and updates to fix validation notebook
```

**Comment by lmz** [comment]
Are there documentation update to go with this?  It would be helpful to have some of the documentation in https://github.com/ActivitySim/populationsim/pull/123 included in the PopulationSim documentation (https://github.com/ActivitySim/populationsim/tree/gh-pages/_sources)

**Comment by jamiecook** [comment]
@lmz - I'm not sure what documentation updates would be relevant? I've been through the docs and I can't really see anywhere where there is any real discussion of the nature of the incidence and the previous requirement that it be binary.... 

If you can point me to a particular section that would be appropriate I'm happy to update though.

**Comment by jamiecook** [comment]
Also... i'm not sure why the build failed... it doesn't seem to have anything to do with this PR

image

**Comment by bstabler** [comment]
@jamiecook - we plan to get to this soon...we're thinking about how the documentation may want to include stuff like this

**Comment by binnympaul** [comment]
> @jamiecook - we plan to get to this soon...we're thinking about how the documentation may want to include stuff like this

Currently, we do not explain the mathematical details of the algorithm in the wiki, but we cite the TRB paper which describes the mathematical equations in detail.  This PR would fix a mismatch between mathematical specification and implementation.

**Comment by jamiecook** [comment]
@binnympaul  - yeah i was confused as to why documentation was mentioned as this just brings the implementation in line with the existing documentation.

**Comment by bstabler** [comment]
I'm pulling this so the code is up-to-date with the paper.  I created #128 for us to add documentation to the user's guide to describe the algorithms/methods.

**Pull request merged**